hashing/hashing.cc

```diff
@@ -28,16 +28,16 @@ pair<ast::ParsedFile, core::UsageHash> rewriteAST(const core::GlobalState &origi
     return make_pair<ast::ParsedFile, core::UsageHash>(move(rewritten), subst.getAllNames());
 }
 
-core::FileHash computeFileHashForAST(unique_ptr<core::GlobalState> &lgs, core::UsageHash usageHash,
-                                     ast::ParsedFile file) {
+unique_ptr<core::FileHash> computeFileHashForAST(unique_ptr<core::GlobalState> &lgs, core::UsageHash usageHash,
+                                                 ast::ParsedFile file) {
     vector<ast::ParsedFile> single;
     single.emplace_back(move(file));
 
     core::Context ctx(*lgs, core::Symbols::root(), single[0].file);
     auto workers = WorkerPool::create(0, lgs->tracer());
     realmain::pipeline::resolve(lgs, move(single), opts(), *workers);
 
-    return {move(*lgs->hash()), move(usageHash)};
+    return make_unique<core::FileHash>(core::FileHash{move(*lgs->hash()), move(usageHash)});
```

Comment:
What do you think about adding a constructor to `core::FileHash` so that we can forward the args here?

Reply:
Works for me.